linux_os/guide/system/logging/rsyslog_sending_messages/rsyslog_remote_tls/tests/missing_option_include_file.fail.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+# remediation = none
```

Comment:
I saw most of the failing test scenarios are not remediating because of the `# remediation = none`  header.
Is that intentional? Why the remediation is not executed in these cases?

Reply:
I am not sure why it is like that. It has been proposed by @jan-cerny  in https://github.com/ComplianceAsCode/content/pull/4788
Jan, do you remember what was the reason to disable remediations during some test scenarios?

Reply:
@jan-cerny , do you remember the context where it was decided to not remediate in some test scenarios? I would be fine to treat this in a separate PR/issue, but it would be good to have this context so we can actually track it.

Reply:
I think that it was to prevent the test errors because at that moment there was no remediation for this rule, because  these scenarios were introduced in 2019 but the remediation was introduced in 2022.

Reply:
Thanks @jan-cerny . It makes sense. It also makes sense to remove the header now. @vojtapolasek , are you comfortable to update this PR?

Reply:
Well, yes, I can update that, but it will probably require more thorough update (maybe rewrite) of remediations. But that's good.